static/js/dashboard.js

```diff
@@ -749,11 +749,15 @@ $(document).ready(function () {
 
   function renderDashboardInfo() {
     var el = document.getElementById("dashboard-info")
+    var depositedText = ""
+    if (state.validatorsCount.deposited > 0) {
+      depositedText = `${state.validatorsCount.deposited} deposited, `
```

Comment:
**Minor:** Can we move the `,` to the front to make it consistent with the slashedText?

```suggestion
      depositedText = `, ${state.validatorsCount.deposited} deposited`
```

Reply:
Sure, done.